packages/@coorpacademy-app-review/src/test/util/services.mock.ts

```diff
@@ -41,6 +42,11 @@ const getSlideFixture = (ref: string): SlideFromAPI => {
   }
 };
 
+const fetchSkillResponse: Skill = {
+  ref: 'skill_NyxtYFYir',
+  name: 'Digital Awareness'
+};
```

Comment:
surement on aura besoin de plus de cas après:
```suggestion
const fetchSkillResponse: Record<string, Skill> = {
  skill_NyxtYFYir: {
    ref: 'skill_NyxtYFYir',
    name: 'Digital Awareness'
  }
};
```